src/stories/Interactive.stories.mdx

```diff
@@ -55,10 +61,31 @@ A TR to represent a draggable person, etc in a list
 <Canvas>
   <Story name="Person TR">
     {() => {
-      const michiel = UI.rdf.namedNode('https://michielbdejong.inrupt.net/profile/card#me')
+      const michiel = UI.rdf.namedNode(
+        'https://michielbdejong.inrupt.net/profile/card#me'
+      )
       return UI.widgets.personTR(document, null, michiel)
-    }
-    }
+    }}
+  </Story>
+</Canvas>
+
+## renderAsDIV
+
+A Div to represent a draggable person, etc in a list
+
+[API documentation](https://solid.github.io/solid-ui/Documentation/api/modules/_widgets_buttons_.html#persontr)
```

Comment:
**issue** The documentation link is wrong, it should not point to persontr. Besides that all those links seem to be broken, perhaps you can update them